hugegraph-dist/src/assembly/static/bin/gremlin-console.sh

```diff
@@ -113,7 +113,7 @@ if [ -z "${HADOOP_GREMLIN_LIBS:-}" ]; then
 fi
 
 if [ -z "${JAVA_OPTIONS:-}" ]; then
-    JAVA_OPTIONS="-Dtinkerpop.ext=$EXT -Dlog4j.configuration=conf/log4j-console.properties -Dgremlin.log4j.level=$GREMLIN_LOG_LEVEL -javaagent:$LIB/jamm-0.3.0.jar"
+    JAVA_OPTIONS="-Dtinkerpop.ext=$EXT -Dlog4j.configurationFile=conf/log4j2.xml -Dgremlin.log4j.level=$GREMLIN_LOG_LEVEL -javaagent:$LIB/jamm-0.3.0.jar"
 fi
```

Comment:
could also echo（print）the options if need（or helpful）

Reply:
@imbajin If set `-l TRACE` or `-l DEBUG`, it will print out the actual running command with options.

```bash
./bin/gremlin-console.sh -l DEBUG -i scripts/example.groovy
```